vic/drivers/shared_image/src/vic_restore.c

```diff
@@ -970,7 +970,7 @@ check_init_state_file(void)
             dimlen = get_nc_dimension(&(filenames.init_state), "lake_node");
             if (dimlen != options.Nlakenode) {
                 log_err("Number of lake nodes in state file does not "
-                        "match LAKE_NODES (%d) in global parameter file",
+                        "match LAKE_NODES (%zu) in global parameter file",
```

Comment:
It's OK for now, but I would have preferred this as a separate PR to keep the individual PRs as focused as possible. Where this would break things is if we had to roll-back a PR for some reason. If we rolled back the `MAX_SNOW_ALBEDO` PR, that would also roll-back this unrelated bug fix. As said, leave this time.